### PR TITLE
[5.8] Correct middleware authenticate phpDoc

### PR DESCRIPTION
**Pull request by MammutAlex**
Add rerurn volid to Authenticate redirectTo

### CODE REVIEW

File: app/Http/Middleware/Authenticate.php [hunk 1]
```diff
@@ -10,7 +10,7 @@ class Authenticate extends Middleware
      * Get the path the user should be redirected to when they are not authenticated.
      *
      * @param  \Illuminate\Http\Request  $request
-     * @return string
+     * @return void|string
```
Comment (driesvints) [inline app/Http/Middleware/Authenticate.php]: Revert these two please.

Comment (MammutAlex) [reply app/Http/Middleware/Authenticate.php]: Сorrected

File: app/Http/Middleware/Authenticate.php [hunk 2]
```diff
@@ -10,7 +10,7 @@ class Authenticate extends Middleware
      * Get the path the user should be redirected to when they are not authenticated.
      *
      * @param  \Illuminate\Http\Request  $request
-     * @return string
+     * @return string|void
```
Comment (bonzai) [inline app/Http/Middleware/Authenticate.php]: It's `string|null` rather than `string|void`.

Comment (MammutAlex) [reply app/Http/Middleware/Authenticate.php]: Why `null`?
That function may not return any value (`void`)

**Pull request closed without merging**